schemas/iso19139.che/src/main/plugin/iso19139.che/index-fields/default.xsl

```diff
@@ -1122,8 +1122,7 @@
                              $positionName, '|',
                              $address, '|',
                              string-join($phone, ','), '|',
-                             $uuid, '|',
-                             $position)}"
+                             $uuid, '|0')}"
```

Comment:
Can we get rid of position variable ?

Reply:
What was the purpose of position ?
https://github.com/fxprunayre/geocat/blob/fad9713a2e8d7a53a71bb537d2f01948ea2897db/schemas/iso19139.che/src/main/plugin/iso19139.che/index-fields/default.xsl#L962

Reply:
Position is used in the Angular view to display contact in the same order as in the record in some client apps. Not needed for geocat.ch but we keep it in GN.